src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -494,8 +494,10 @@
                                                     (on-long-press
                                                      (concat
                                                       [{:on-press #(re-frame/dispatch [:chat.ui/reply-to-message message])
+                                                        :id       :reply
```

Comment:
last time it was string :)

Reply:
nope, it was a string only the first time :) I changed it to keyword in that PR